assets/js/app.js

```diff
@@ -0,0 +1,21 @@
+var templateString = document.querySelector('main-template').innerHTML;
```

Comment:
To grab your template, your query string needs to be `#main-template` to grab the template by its `id`. Just like we would create a selector when doing CSS.